vedic_maths/folding.py

```diff
@@ -223,14 +229,14 @@ def construct(self):
         ]
         e = Explanation(self, text, wait=2, fade=True, aligned_edge=LEFT)
 
-        """ text = [
+        text = [
```

Comment:
This is confusing, since we don't have an example; that is why I commented it out

Reply:
Noted.

Reply:
Please push this change and I will merge. Everything else is fine.

Reply:
Pushed it. Not able to create a new pull request.